yarn.lock

```diff
@@ -0,0 +1,3549 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
did you add this file on purpose?

Reply:
no. apparently yarn was never used before
will remove it